pkgs/tools/misc/jfrog-cli/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildGoModule, fetchFromGitHub, pkgs, lib }:
+
+buildGoModule rec {
+  pname = "jfrog-cli";
+  version = "2.32.0";
+  vendorSha256 = "sha256-nL+2Yc4gI2+SoxoaGlazecsrcVkVh6Ig9sqITSOa5e0=";
+
+  src = fetchFromGitHub {
+    owner = "jfrog";
+    repo = "jfrog-cli";
+    rev = "v${version}";
+    sha256 = "sha256-EyDX4OrBAzc5eYR660SrGIG61TRlWnnV/GAtXy7DfEI=";
+  };
+
+  # The binary is static, so no need for patchelf
+  dontPatchELF = true;
+
+  nativeBuildInputs = [ pkgs.glibc.static ];
+  CGO_ENABLED = 0;
+  ldflags = [
+    "-w"
+    "-extldflags"
+    "-static"
+  ];
```

Comment:
@Detegr What is the reason to make the binary static? I don't see a benefit in doing this in nixpkgs.

Reply:
That's just how it is built and distributed upstream. I'm new to Nix and I didn't think it through. I'll try building a dynamic version tomorrow and will update the PR if there's no issues.